### ISSUE TITLE
Not loading to GC menu without sd card 

### ISSUE THREAD

**Issue by antoxa2584x** [webhdx/PicoBoot#121]
| Answer some general questions first                             	|  	|
|:-------------------------------------------------------------------	|:------:	|
| Are you my GitHub sponsor? | no 	|
| Did you check older issues to see if it was requested before? | yes	|

On previous firmware where I boot GameCube without micro sd card, GameCube booted to oem menu. 

On current 0.4 If no micro sd in sp2sd slot, GameCube stuck on no ipl.dol screen

**Comment by webhdx** [comment]
Yes this was changed in https://github.com/redolution/gekkoboot/ which PicoBoot uses as its default payload. I believe right now you can boot into stock menu by holding D-Pad Left or Reset button.
